scalableminds/webknossos
# Volume Fixes

Don't crash when annotating volume buckets for the first time and having fallback rendering enabled; also properly log errors in save saga

### URL of deployed dev instance (used for testing):
- https://fixvolumesaving.webknossos.xyz

### Steps to test:
- create a new volume (or hybrid) annotation **without fallback data**
- ensure "Render Missing Data Black" is disabled
- Brush & Save ==> On master this fails, on this branch this should work

### Issues:
- follow-up to #5733 

------
